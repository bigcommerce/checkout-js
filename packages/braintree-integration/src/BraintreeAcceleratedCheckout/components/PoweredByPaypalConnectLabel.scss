.powered-by-paypal-connect-label {
  font-size: 0.77rem;
  margin: 0.77rem 0 0;

  & > .icon {
    margin: 0 0.5em;
    width: 4.24rem;
    height: 0.77rem;
  }
}
