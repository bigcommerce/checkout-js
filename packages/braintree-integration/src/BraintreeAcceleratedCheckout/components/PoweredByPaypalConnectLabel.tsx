import React from 'react';

import { IconPayPalConnect } from '@bigcommerce/checkout/ui';

import './PoweredByPaypalConnectLabel.scss';

const PoweredByPaypalConnectLabel = () => (
    <div className="powered-by-paypal-connect-label">
        Powered by
        <IconPayPalConnect />
    </div>
);

export default PoweredByPaypalConnectLabel;
