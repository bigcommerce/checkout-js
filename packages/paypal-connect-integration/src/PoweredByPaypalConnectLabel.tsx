import React from 'react';

import { TranslatedString } from '@bigcommerce/checkout/locale';
import { IconPayPalConnect } from '@bigcommerce/checkout/ui';

import './PoweredByPaypalConnectLabel.scss';

const PoweredByPaypalConnectLabel = () => (
    <div className="powered-by-paypal-connect">
        <div className="powered-by-paypal-connect-label">
            <TranslatedString id="remote.powered_by" />
            <IconPayPalConnect />
        </div>
    </div>
);

export default PoweredByPaypalConnectLabel;
