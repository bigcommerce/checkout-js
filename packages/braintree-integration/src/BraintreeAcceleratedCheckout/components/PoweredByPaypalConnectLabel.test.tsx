import { render } from '@testing-library/react';
import React from 'react';

import PoweredByPaypalConnectLabel from './PoweredByPaypalConnectLabel';

describe('PoweredByPaypalConnectLabel', () => {
    it('renders PayPal Connect label', () => {
        const view = render(<PoweredByPaypalConnectLabel />);

        expect(view).toMatchSnapshot();
    });
});
