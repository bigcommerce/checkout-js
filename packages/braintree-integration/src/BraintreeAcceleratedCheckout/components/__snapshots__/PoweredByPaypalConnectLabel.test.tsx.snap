// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PoweredByPaypalConnectLabel renders PayPal Connect label 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="powered-by-paypal-connect-label"
      >
        Powered by
        <div
          class="icon"
        >
          <svg
            fill="none"
            height="12"
            viewBox="0 0 70 12"
            width="70"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M2.75845 7.01162V3.64258H1.44776C0.64798 3.64258 0 4.38005 0 5.29085V9.99776C0 10.9086 0.64798 11.646 1.44776 11.646H8.02498C8.82476 11.646 9.47274 10.9086 9.47274 9.99776V8.65989H4.2062C3.40643 8.65989 2.75845 7.92128 2.75845 7.01162Z"
              fill="#313131"
            />
            <path
              d="M10.7825 0.655762H4.20533C3.40555 0.655762 2.75757 1.39323 2.75757 2.30403V3.64191H8.0241C8.82388 3.64191 9.47186 4.37938 9.47186 5.29018V8.65922H10.7825C11.5823 8.65922 12.2303 7.92174 12.2303 7.01095V2.30403C12.2303 1.39437 11.5823 0.655762 10.7825 0.655762Z"
              fill="#5B5B5B"
            />
            <path
              d="M8.02508 3.64258H2.75854V7.01162C2.75854 7.92242 3.40653 8.65989 4.2063 8.65989H9.47284V5.29085C9.4717 4.38005 8.82372 3.64258 8.02508 3.64258Z"
              fill="#DBDBDB"
            />
            <path
              d="M15.0747 5.57183C15.0747 2.61287 17.0379 0.354004 20.0682 0.354004C22.3124 0.354004 23.8689 1.50383 24.4863 3.20081C24.4999 3.24272 24.5146 3.29937 24.5146 3.34128C24.5146 3.48175 24.4024 3.56558 24.2201 3.56558H22.9298C22.7474 3.56558 22.6352 3.49535 22.5514 3.32769C22.0609 2.43049 21.1909 1.88333 20.0546 1.88333C18.0631 1.88333 16.9133 3.49648 16.9133 5.57183C16.9133 7.6619 18.1198 9.24673 20.1113 9.24673C21.303 9.24673 22.0745 8.64407 22.5661 7.8998C22.6647 7.74573 22.7768 7.6755 22.9445 7.6755H24.1645C24.3469 7.6755 24.4591 7.73214 24.4591 7.8862C24.4591 7.92812 24.4455 7.98476 24.4172 8.055C23.8145 9.47104 22.3135 10.7761 20.041 10.7761C17.0526 10.7761 15.0747 8.54551 15.0747 5.57183Z"
              fill="#252525"
            />
            <path
              d="M25.2576 6.90474C25.2576 4.59037 26.7869 2.96362 29.1001 2.96362C31.4281 2.96362 32.9574 4.59037 32.9574 6.90474C32.9574 9.1772 31.4564 10.804 29.0718 10.804C26.7019 10.804 25.2576 9.16361 25.2576 6.90474ZM31.1483 6.89115C31.1483 5.43206 30.3632 4.39439 29.0729 4.39439C27.7962 4.39439 27.0112 5.43206 27.0112 6.89115C27.0112 8.39215 27.7826 9.37319 29.0729 9.37319C30.3768 9.37432 31.1483 8.39215 31.1483 6.89115Z"
              fill="#252525"
            />
            <path
              d="M34.191 10.2569V3.53918C34.1775 3.32847 34.2896 3.21632 34.4992 3.21632H35.4949C35.7056 3.21632 35.8178 3.32847 35.8178 3.53918V3.66492C35.8178 3.81899 35.8042 3.97305 35.8042 4.12825C35.8042 4.14185 35.8042 4.14185 35.8178 4.14185C36.2664 3.46894 36.9823 2.97729 38.1174 2.97729C39.8008 2.97729 40.7819 4.11353 40.7819 6.00649V10.2557C40.7819 10.4664 40.6697 10.5786 40.459 10.5786H39.393C39.1823 10.5786 39.0702 10.4664 39.0702 10.2557V6.77681C39.0702 5.33245 38.719 4.40693 37.5692 4.40693C36.4046 4.40693 35.9005 5.31886 35.9005 6.46868V10.2557C35.9005 10.4664 35.7883 10.5786 35.5776 10.5786H34.5116C34.3032 10.5797 34.191 10.4676 34.191 10.2569Z"
              fill="#252525"
            />
            <path
              d="M42.3959 10.2569V3.53918C42.3823 3.32847 42.4944 3.21632 42.704 3.21632H43.6998C43.9105 3.21632 44.0226 3.32847 44.0226 3.53918V3.66492C44.0226 3.81899 44.009 3.97305 44.009 4.12825C44.009 4.14185 44.009 4.14185 44.0226 4.14185C44.4712 3.46894 45.1872 2.97729 46.3223 2.97729C48.0057 2.97729 48.9867 4.11353 48.9867 6.00649V10.2557C48.9867 10.4664 48.8745 10.5786 48.6638 10.5786H47.5978C47.3871 10.5786 47.275 10.4664 47.275 10.2557V6.77681C47.275 5.33245 46.9238 4.40693 45.774 4.40693C44.6094 4.40693 44.1053 5.31886 44.1053 6.46868V10.2557C44.1053 10.4664 43.9932 10.5786 43.7825 10.5786H42.7165C42.508 10.5797 42.3959 10.4676 42.3959 10.2569Z"
              fill="#252525"
            />
            <path
              d="M50.1392 6.87716C50.1392 4.63302 51.6538 2.96436 53.8979 2.96436C56.2542 2.96436 57.5026 4.71798 57.5026 6.86356V7.00403C57.5026 7.21474 57.3621 7.32689 57.1661 7.32689L51.8509 7.3133C51.963 8.51976 52.7764 9.38864 53.9693 9.38864C54.6841 9.38864 55.1893 9.12243 55.5122 8.68742C55.6243 8.54695 55.7365 8.47672 55.9053 8.47672H56.9713C57.1537 8.47672 57.2794 8.56054 57.2794 8.67269C57.2794 8.71461 57.2794 8.75652 57.1808 8.95364C56.6756 9.96412 55.5677 10.8047 53.9273 10.8047C51.6254 10.8036 50.1392 9.1768 50.1392 6.87716ZM55.7909 6.09097C55.6787 5.08162 54.9775 4.29543 53.8832 4.29543C52.8172 4.29543 52.0457 4.92642 51.9052 6.09097H55.7909Z"
              fill="#252525"
            />
            <path
              d="M58.189 6.89115C58.189 4.64701 59.69 2.96362 61.976 2.96362C63.7296 2.96362 64.8795 3.91747 65.3281 5.29159C65.3417 5.33351 65.3564 5.39015 65.3564 5.43206C65.3564 5.58613 65.2306 5.65636 65.0483 5.65636H63.912C63.7296 5.65636 63.6175 5.57253 63.5337 5.40374C63.2255 4.78635 62.7339 4.39326 61.9488 4.39326C60.7152 4.39326 59.9437 5.43093 59.9437 6.8617C59.9437 8.37629 60.7152 9.38678 62.0191 9.38678C62.7622 9.38678 63.3094 9.02201 63.6175 8.46126C63.7013 8.30719 63.8135 8.22336 63.9959 8.22336H65.0199C65.2023 8.22336 65.3428 8.2936 65.3428 8.41934C65.3428 8.46125 65.3292 8.50317 65.2861 8.61532C64.8228 9.84898 63.7013 10.8028 61.9477 10.8028C59.6197 10.804 58.189 9.14888 58.189 6.89115Z"
              fill="#252525"
            />
            <path
              d="M66.9131 8.72776V4.4921H66.128C65.9173 4.4921 65.8052 4.37995 65.8052 4.16924V3.53825C65.8052 3.32755 65.9173 3.2154 66.128 3.2154H66.9131V1.54673C66.9131 1.33603 67.0252 1.22388 67.2359 1.22388H68.3019C68.5126 1.22388 68.6248 1.33603 68.6248 1.54673V3.2154H69.6772C69.8879 3.2154 70.0001 3.32755 70.0001 3.53825V4.16924C70.0001 4.37995 69.8879 4.4921 69.6772 4.4921H68.6248V8.46154C68.6248 9.14917 68.8355 9.37347 69.3396 9.37347C69.4801 9.37347 69.522 9.35987 69.6206 9.35987C69.8029 9.35987 69.9015 9.4437 69.9015 9.69633V10.3556C69.9015 10.7057 69.6489 10.7204 69.2558 10.7204C67.7695 10.7193 66.9131 10.2707 66.9131 8.72776Z"
              fill="#252525"
            />
          </svg>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="powered-by-paypal-connect-label"
    >
      Powered by
      <div
        class="icon"
      >
        <svg
          fill="none"
          height="12"
          viewBox="0 0 70 12"
          width="70"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M2.75845 7.01162V3.64258H1.44776C0.64798 3.64258 0 4.38005 0 5.29085V9.99776C0 10.9086 0.64798 11.646 1.44776 11.646H8.02498C8.82476 11.646 9.47274 10.9086 9.47274 9.99776V8.65989H4.2062C3.40643 8.65989 2.75845 7.92128 2.75845 7.01162Z"
            fill="#313131"
          />
          <path
            d="M10.7825 0.655762H4.20533C3.40555 0.655762 2.75757 1.39323 2.75757 2.30403V3.64191H8.0241C8.82388 3.64191 9.47186 4.37938 9.47186 5.29018V8.65922H10.7825C11.5823 8.65922 12.2303 7.92174 12.2303 7.01095V2.30403C12.2303 1.39437 11.5823 0.655762 10.7825 0.655762Z"
            fill="#5B5B5B"
          />
          <path
            d="M8.02508 3.64258H2.75854V7.01162C2.75854 7.92242 3.40653 8.65989 4.2063 8.65989H9.47284V5.29085C9.4717 4.38005 8.82372 3.64258 8.02508 3.64258Z"
            fill="#DBDBDB"
          />
          <path
            d="M15.0747 5.57183C15.0747 2.61287 17.0379 0.354004 20.0682 0.354004C22.3124 0.354004 23.8689 1.50383 24.4863 3.20081C24.4999 3.24272 24.5146 3.29937 24.5146 3.34128C24.5146 3.48175 24.4024 3.56558 24.2201 3.56558H22.9298C22.7474 3.56558 22.6352 3.49535 22.5514 3.32769C22.0609 2.43049 21.1909 1.88333 20.0546 1.88333C18.0631 1.88333 16.9133 3.49648 16.9133 5.57183C16.9133 7.6619 18.1198 9.24673 20.1113 9.24673C21.303 9.24673 22.0745 8.64407 22.5661 7.8998C22.6647 7.74573 22.7768 7.6755 22.9445 7.6755H24.1645C24.3469 7.6755 24.4591 7.73214 24.4591 7.8862C24.4591 7.92812 24.4455 7.98476 24.4172 8.055C23.8145 9.47104 22.3135 10.7761 20.041 10.7761C17.0526 10.7761 15.0747 8.54551 15.0747 5.57183Z"
            fill="#252525"
          />
          <path
            d="M25.2576 6.90474C25.2576 4.59037 26.7869 2.96362 29.1001 2.96362C31.4281 2.96362 32.9574 4.59037 32.9574 6.90474C32.9574 9.1772 31.4564 10.804 29.0718 10.804C26.7019 10.804 25.2576 9.16361 25.2576 6.90474ZM31.1483 6.89115C31.1483 5.43206 30.3632 4.39439 29.0729 4.39439C27.7962 4.39439 27.0112 5.43206 27.0112 6.89115C27.0112 8.39215 27.7826 9.37319 29.0729 9.37319C30.3768 9.37432 31.1483 8.39215 31.1483 6.89115Z"
            fill="#252525"
          />
          <path
            d="M34.191 10.2569V3.53918C34.1775 3.32847 34.2896 3.21632 34.4992 3.21632H35.4949C35.7056 3.21632 35.8178 3.32847 35.8178 3.53918V3.66492C35.8178 3.81899 35.8042 3.97305 35.8042 4.12825C35.8042 4.14185 35.8042 4.14185 35.8178 4.14185C36.2664 3.46894 36.9823 2.97729 38.1174 2.97729C39.8008 2.97729 40.7819 4.11353 40.7819 6.00649V10.2557C40.7819 10.4664 40.6697 10.5786 40.459 10.5786H39.393C39.1823 10.5786 39.0702 10.4664 39.0702 10.2557V6.77681C39.0702 5.33245 38.719 4.40693 37.5692 4.40693C36.4046 4.40693 35.9005 5.31886 35.9005 6.46868V10.2557C35.9005 10.4664 35.7883 10.5786 35.5776 10.5786H34.5116C34.3032 10.5797 34.191 10.4676 34.191 10.2569Z"
            fill="#252525"
          />
          <path
            d="M42.3959 10.2569V3.53918C42.3823 3.32847 42.4944 3.21632 42.704 3.21632H43.6998C43.9105 3.21632 44.0226 3.32847 44.0226 3.53918V3.66492C44.0226 3.81899 44.009 3.97305 44.009 4.12825C44.009 4.14185 44.009 4.14185 44.0226 4.14185C44.4712 3.46894 45.1872 2.97729 46.3223 2.97729C48.0057 2.97729 48.9867 4.11353 48.9867 6.00649V10.2557C48.9867 10.4664 48.8745 10.5786 48.6638 10.5786H47.5978C47.3871 10.5786 47.275 10.4664 47.275 10.2557V6.77681C47.275 5.33245 46.9238 4.40693 45.774 4.40693C44.6094 4.40693 44.1053 5.31886 44.1053 6.46868V10.2557C44.1053 10.4664 43.9932 10.5786 43.7825 10.5786H42.7165C42.508 10.5797 42.3959 10.4676 42.3959 10.2569Z"
            fill="#252525"
          />
          <path
            d="M50.1392 6.87716C50.1392 4.63302 51.6538 2.96436 53.8979 2.96436C56.2542 2.96436 57.5026 4.71798 57.5026 6.86356V7.00403C57.5026 7.21474 57.3621 7.32689 57.1661 7.32689L51.8509 7.3133C51.963 8.51976 52.7764 9.38864 53.9693 9.38864C54.6841 9.38864 55.1893 9.12243 55.5122 8.68742C55.6243 8.54695 55.7365 8.47672 55.9053 8.47672H56.9713C57.1537 8.47672 57.2794 8.56054 57.2794 8.67269C57.2794 8.71461 57.2794 8.75652 57.1808 8.95364C56.6756 9.96412 55.5677 10.8047 53.9273 10.8047C51.6254 10.8036 50.1392 9.1768 50.1392 6.87716ZM55.7909 6.09097C55.6787 5.08162 54.9775 4.29543 53.8832 4.29543C52.8172 4.29543 52.0457 4.92642 51.9052 6.09097H55.7909Z"
            fill="#252525"
          />
          <path
            d="M58.189 6.89115C58.189 4.64701 59.69 2.96362 61.976 2.96362C63.7296 2.96362 64.8795 3.91747 65.3281 5.29159C65.3417 5.33351 65.3564 5.39015 65.3564 5.43206C65.3564 5.58613 65.2306 5.65636 65.0483 5.65636H63.912C63.7296 5.65636 63.6175 5.57253 63.5337 5.40374C63.2255 4.78635 62.7339 4.39326 61.9488 4.39326C60.7152 4.39326 59.9437 5.43093 59.9437 6.8617C59.9437 8.37629 60.7152 9.38678 62.0191 9.38678C62.7622 9.38678 63.3094 9.02201 63.6175 8.46126C63.7013 8.30719 63.8135 8.22336 63.9959 8.22336H65.0199C65.2023 8.22336 65.3428 8.2936 65.3428 8.41934C65.3428 8.46125 65.3292 8.50317 65.2861 8.61532C64.8228 9.84898 63.7013 10.8028 61.9477 10.8028C59.6197 10.804 58.189 9.14888 58.189 6.89115Z"
            fill="#252525"
          />
          <path
            d="M66.9131 8.72776V4.4921H66.128C65.9173 4.4921 65.8052 4.37995 65.8052 4.16924V3.53825C65.8052 3.32755 65.9173 3.2154 66.128 3.2154H66.9131V1.54673C66.9131 1.33603 67.0252 1.22388 67.2359 1.22388H68.3019C68.5126 1.22388 68.6248 1.33603 68.6248 1.54673V3.2154H69.6772C69.8879 3.2154 70.0001 3.32755 70.0001 3.53825V4.16924C70.0001 4.37995 69.8879 4.4921 69.6772 4.4921H68.6248V8.46154C68.6248 9.14917 68.8355 9.37347 69.3396 9.37347C69.4801 9.37347 69.522 9.35987 69.6206 9.35987C69.8029 9.35987 69.9015 9.4437 69.9015 9.69633V10.3556C69.9015 10.7057 69.6489 10.7204 69.2558 10.7204C67.7695 10.7193 66.9131 10.2707 66.9131 8.72776Z"
            fill="#252525"
          />
        </svg>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
